appveyor.yml

```diff
@@ -35,7 +42,7 @@ build_script:
 
     xcopy C:\projects\pmmpthread C:\projects\php-src\ext\pmmpthread /s /e /y /q
 
-    phpsdk_deps -u -t %VC_VER% -b %PHP_VER% -a %ARCH% -f -d C:\projects\php-src\deps
+    phpsdk_deps -u -t %VC_VER% -b %PHP_VER% -s %PHP_STABILITY% -a %ARCH% -f -d C:\projects\php-src\deps

```

Comment:
PHP 8.4 stable is still not released at https://windows.php.net/downloads/php-sdk/deps/series/ , so I added -s PHP_STABILITY.